enacts/onset/maproom.py

```diff
@@ -664,10 +663,9 @@ def onset_tile(tz, tx, ty):
 )
 def set_colorbar(search_start_day, search_start_month, search_days, map_choice):
     if map_choice == "pe":
-        thresholds = pingrid.CORRELATION_COLORMAP_KEY
         return (
             f"Probabily of onset date to be {search_days} past {search_start_day} {search_start_month}",
-            pingrid.to_dash_colorscale(pingrid.CORRELATION_COLORMAP, thresholds=thresholds),
```

Comment:
Why is this no longer needed?

Reply:
I think I had implemented this to have dynamic colorscales (which are needed for the flexible forecast) but we walked away from that for another reason but when walking away, it seems I missed out removing these.

We still need some sort of dynamic colorscale capacity. See Issue #151 . But these pieces are not part of it. Or if they will be, they'll come back but they don't belong here and now.